### PR TITLE
Models phase2

### PR DESCRIPTION
**Pull request by Antonior26**

### CODE REVIEW

**Comment by Antonior26** [comment]
Do NOT merge please

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+`ReportEvent`
+* `actions` added to `ReportEvent`
+* `consequenceTypes` added to `ReportEvent`
+* `panelName` and `panelVersion` removed from `ReportEvent`, replaced by
+`genePanel`
+* `modeOfInheritance` in `ReportEvent` is now nullable
+* Added `roleInCancer` to `ReportEvent` (nullable)
+* `penetrance` in `ReportEvent` is now nullable
+* `score` in `ReportEvent` is now nullable
+* `genomicFeature` in `ReportEvent` is now an array
```
Comment (greglever) [inline CHANGES.md]: is this an array of genomic features ? Could we rename to `genomicFeatures` if this is the case ?

Comment (Antonior26) [reply CHANGES.md]: done

File: schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/CommonInterpreted.avdl [hunk 2]
```diff
@@ -130,18 +168,42 @@ It is a representation of the zygosity
 
     }
 
-    /**
-    This is the classification of the variant according to standard practice guidelines (e.g. ACMG)
-    */
-    // TODO: refactor with ACMGClassification in ExitQuestionnaire
-    enum VariantClassification {pathogenic_variant, likely_pathogenic_variant, variant_of_unknown_clinical_significance, likely_benign_variant,  benign_variant, not_assessed}
+    record GenePanel{
```
Comment (cjohnson12) [inline schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/CommonInterpreted.avdl]: Would it be such that the record GenePanel can be union {null, record} but all items within a record must be present (therefore you always get a GenePanel name and version in each record in the payload)

Comment (Antonior26) [reply schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/CommonInterpreted.avdl]: Changed

File: schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/CommonInterpreted.avdl [hunk 3]
```diff
@@ -248,101 +301,81 @@ It is a representation of the zygosity
...
 
         /**
-        alternate: Alternate Allele sequence, the same provided in vcf
+        array of genotypes for the samples
         */
-        string alternate;
-
-
-        array<CalledGenotype> calledGenotypes;
+        array<VariantCall> variantCalls;
```
Comment (cjohnson12) [inline schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/CommonInterpreted.avdl]: If VariantCalls is in CommonInterpreted, is there a "na" for somatic variants?

Comment (Antonior26) [reply schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/CommonInterpreted.avdl]: Nop,  VariantCalls is still valid for somatic variants zygosity (inside VariantCalls) is the one that will be na

File: builds.json [hunk 4]
```diff
@@ -1,6 +1,38 @@
 {
   "builds" :
   [
+    {
+      "version": "7.0.0",
```
Comment (priesgo) [inline builds.json]: Just as a side note. Once CVA is adapted to these latest models, we don't need build version 7.0.0. We can stick with 6.0.0

**Pull request merged**